<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1489</width>
    <height>789</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QLabel" name="Title">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(189, 255, 126);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="text">
      <string>Dynamic waiting time analysis</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="3">
    <widget class="PlotWidget" name="current_plot">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="DoIVCVframe">
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>300</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame { background : rgb(200, 200, 200) }  </string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="4">
       <widget class="QFrame" name="IV_settings_settings">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame { background :rgb(144, 144, 144) }</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="2">
          <widget class="QLabel" name="voltages_steps_label">
           <property name="text">
            <string>Voltage steps [dV]</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QDoubleSpinBox" name="max_voltage_IV">
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="minimum">
            <double>-1000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>0.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>10.000000000000000</double>
           </property>
           <property name="value">
            <double>-200.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="complience_IV">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>-1000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>0.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>-50.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="complience_label_IV">
           <property name="text">
            <string>Complience [uA]</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QDoubleSpinBox" name="voltage_steps_IV">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::UpDownArrows</enum>
           </property>
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
           <property name="value">
            <double>10.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="max_voltage_label_IV">
           <property name="text">
            <string>Maximum bias voltage [V]</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="StartButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="StopButton">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Output file:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="output_file"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="change_directory">
        <property name="text">
         <string>Output directory</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QLineEdit" name="output_directory">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="4">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="4">
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
